src/core/dbus-mount.c

```diff
@@ -34,6 +41,27 @@ static const char *mount_get_fstype(const Mount *m) {
         return NULL;
 }
 
+static int bus_mount_method_remount(sd_bus_message *message, void *userdata, sd_bus_error *error) {
+
+        _cleanup_(sd_bus_creds_unrefp) sd_bus_creds *creds = NULL;
+        _cleanup_set_free_ Set *pids = NULL;
+        Mount *m = userdata;
+        Unit *u = UNIT(m);
+        int r;
+
+        assert(message);
+
+        r = bus_mount_set_transient_property(m, "Options", message, UNIT_RUNTIME, error);
```

Comment:
Maybe, `bus_set_transient_string()` is enough?

Reply:
ok